app/controllers/search_controller.rb

```diff
@@ -31,6 +31,10 @@ def tags
     @tags = ExecuteSearch.new.by(:tags, @search_criteria).paginate(page: params[:page], per_page: 20)
   end
 
+  def all_content
+    @nodes = ExecuteSearch.new.by(:all, @search_criteria)
```

Comment:
Also i see the pagination is not correct here . Pagination is a 2- step approach : 
Example : 1.)  see **.paginate** here - 
https://github.com/publiclab/plots2/blob/9724d9ad8563b2fc05127193df8410d546f569bb/app/controllers/search_controller.rb#L12

2.) And then using same variable **@wiki** here - 
 https://github.com/publiclab/plots2/blob/9724d9ad8563b2fc05127193df8410d546f569bb/app/views/wiki/_wikis.html.erb#L22

Documentation : https://github.com/mislav/will_paginate/wiki

### Solution : 

* Use Pagination line in only partial views (i.e wiki , notes , questions , etc.)  and not in **all_content.html.erb**
*  Individually paginate content of @nodes i.e @nodes[wiki].paginate(page: params[:page], per_page: 24) , etc. 


 @publiclab/mentors --- can anyone verify if this solution is the best solution here ? 
Thanks!

Reply:
Thanks @sagarpreet-chadha

Reply:
Feel free to ping @publiclab/reviewers if you want help somewhere 😄 !